.mci.yml

```diff
@@ -82,6 +82,7 @@ tasks:
                   make
                   make run-examples
                   ./src/mongocxx/test/test_driver
+                  ./src/bsoncxx/test/test_bson
```

Comment:
I'd change the order here. Run the bson tests first, then the driver tests, and then the examples last. That way if you break BSON you will get an error in the bson tests, rather than in the driver tests, etc. Rougly dependency order.